cicecore/drivers/nuopc/cmeps/ice_prescribed_mod.F90

```diff
@@ -211,6 +207,9 @@ subroutine ice_prescribed_init(clock, mesh, rc)
           hin_max(1) = 999._dbl_kind
        end if
 
+       ! If need initial cice values for coupling
```

Comment:
Do you know this is compatible w/ CESM's use of prescribed ice?

Reply:
W/o knowing this has been tested and confirmed to work for CESM, this should be enclosed w/ ifndef CESMCOUPLED. We should always err on the side of not impacting each other's uses.